modules/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>hibernate-validator-parent</artifactId>
+        <groupId>org.hibernate</groupId>
+        <version>6.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>hibernate-validator-modules</artifactId>
+
+    <name>Hibernate Validator WildFly Module</name>
+    <description>Hibernate Validator modules creation to use in WildFly</description>
+
+    <properties>
+        <module.xml.basedir>${basedir}/src/main/modules</module.xml.basedir>
+        <hibernate.validator.module.slot>${project.version}</hibernate.validator.module.slot>
```

Comment:
That looks good. For ORM and HSEARCH we are using the alias mechanism in addition, allowing to use "6" or "6.1" when referencing the module, making it a bit simpler for the user. Would be nice to have the same here (you can find examples for how it's done in these other projects).